### DISCUSSION TITLE
Critical bug: The settings for Android ID/Device IMEI/GPS Mocking

### DISCUSSION THREAD

**Discussion by mary-ng5519** [Genymobile/feedback#38] [Genymotion Desktop]
Dear team, please help fix device preference setting bugs, currently, the settings for each device cannot last/get saved after devices get cold shutdown (via ending process/close emulator window/not-corresponding cases)
Version 3.3.1 | Revision 20221102-b65230d63
Tons of thanks
p/s: please help move this ticket to the right place/responsible people

**Comment by jmaneyrol69** [comment]
Hi @phathuynh5519,

We would need more information in order to investigate. Could you detail the following:

- what Android version do you use?
- what is your OS (Windows, Linux, macOS) and which version?
- the exact steps to reproduce the bug

Thank you.
